cmd/spot/main.go

```diff
@@ -198,7 +200,17 @@ func runAdHoc(ctx context.Context, targets []string, r *runner.Process) error {
 
 // runGen generates a destination report for the task's targets
```

Comment:
nit: "task's targets" to "tasks' target"

Reply:
Thanks for your input. Fixed.